client/components/ToolBar.tsx

```diff
@@ -91,6 +90,7 @@ export default function ToolBar({
                             }
                         }}
                     />
+                    <h3 className="settingsTab">Settings:</h3>
```

Comment:
I think it looks better without this in here. The individual pane's have headings in them.

Reply:
Deleted.